### PR TITLE
add custom easyblock for Stata

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyblocks/s/stata.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+##
+# Copyright 2009-2017 Ghent University
```
Comment (wpoely86) [inline easybuild/easyblocks/s/stata.py]: you seems to live in the past? 😉

File: easybuild/easyblocks/s/stata.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    def sanity_check_step(self):
+        """Custom sanity check for Stata."""
+        custom_paths = {
+            'files': ['stata', 'xstata'],
+            'dirs': [],
+        }
+        super(EB_Stata, self).sanity_check_step(custom_paths=custom_paths)
+
+        # make sure required libpng library is there for stata
```
Comment (wpoely86) [inline easybuild/easyblocks/s/stata.py]: why do this? It's not a dependency?

Comment (boegel) [reply easybuild/easyblocks/s/stata.py]: Stata requires a very old version of `libpng`, see https://github.com/easybuilders/easybuild-easyconfigs/pull/5102

File: easybuild/easyblocks/s/stata.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing Stata, implemented as an easyblock
+"""
```
Comment (wpoely86) [inline easybuild/easyblocks/s/stata.py]: an author tag?

File: easybuild/easyblocks/s/stata.py (hunk 2 above)
Comment (wpoely86) [reply easybuild/easyblocks/s/stata.py]: Add as comment to the code?

**Pull request merged**